java/runtime/src/main/java/io/ray/runtime/object/ObjectSerializer.java

```diff
@@ -51,6 +51,7 @@
   public static final byte[] OBJECT_METADATA_TYPE_JAVA = "JAVA".getBytes();
   public static final byte[] OBJECT_METADATA_TYPE_PYTHON = "PYTHON".getBytes();
   public static final byte[] OBJECT_METADATA_TYPE_RAW = "RAW".getBytes();
+  public static final byte[] OBJECT_METADATA_TYPE_ARROW = "ARROW".getBytes();
```

Comment:
So we only hold the object in Java? If so, I think this line is unnecessary.

Reply:
And  you only support that putraw in Python right?

Reply:
I plan to support writing/reading in both python and java. I'll push the code for java writing soon.